scripts/createReleasePipeline/create-release-pipeline.sh

```diff
@@ -43,8 +48,19 @@ IMAGENAME="azurerig"
 IMAGENAME=$(echo "$IMAGENAME" | awk '{print tolower($0)}')
 APPNAME=$BASEAPPNAME$STAGE
 RESOURCEGROUPNAME=$BASERESOURCEGROUPNAME$STAGE
-sed -i'' -e " s|\${serviceConnectionId}|$SERVICECONNECTIONID|g; s|\${resourceGroupName}|$RESOURCEGROUPNAME|g; s|\${location}|$LOCATION|g; s|\${registryName}|$REGISTRYNAME|g; s|\${appName}|$APPNAME|g; s|\${registrySku}|$REGISTRYSKU|g; s|\${imageName}|$IMAGENAME|g; s|\${orgName}|$ORGNAME|g; s|\${pipelineName}|$PIPELINENAME|g; s|\${pipelineId}|$SOURCEPIPELINEID|g" $DATAPATH/$DATAFILE
+STAGE_LOWERCASE=$(echo "$STAGE" | awk '{print tolower($0)}')
+COMMON_STORAGEACCOUNT_CONTAINER_NAME="$STAGE_LOWERCASE-test-results"
+COMMON_STORAGEACCOUNT_CONTAINER_URL="https://${COMMON_STORAGEACCOUNT_NAME}.blob.core.windows.net/${COMMON_STORAGEACCOUNT_CONTAINER_NAME}"
+APPNAME_LOWERCASE=$(echo "$APPNAME" | awk '{print tolower($0)}')
+SLACKBOT_HELLO_URL="https://${APPNAME_LOWERCASE}.azurewebsites.net/"
```

Comment:
can we change the name of this variable "APP_BASE_URL" or somethign like that?

Reply:
good catch